### PR TITLE
[OEL-1255] OEL-1280: Remove most of paragraphs submodule (for move to oe_whitelabel)

### PR DESCRIPTION
**Pull request by abel-santos-corral**
Jira issue(s):
- [OEL-1280](https://citnet.tech.ec.europa.eu/CITnet/jira/projects/OEL/issues/OEL-1280)

Related PR in oe_whitelabel: [93](https://github.com/openeuropa/oe_whitelabel/pull/93)

### CODE REVIEW

File: modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.info.yml [hunk 1]
```diff
@@ -1,21 +0,0 @@
-name: OpenEuropa Bootstrap Theme Paragraphs
```
Comment (drishu) [inline modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.info.yml]: you can't delete this file
```
[error]  OpenEuropa Bootstrap Theme Paragraphs is required if OpenEuropa Paragraphs is 
enabled.
 
 [error]   (Currently using Missing or invalid module The following module is marked as installed in the core.extension 
configuration, but it is missing:
```
just delete the config_devel from here

**Review by drishu — CHANGES_REQUESTED** [review]
I will double check that all files are removed (test and whatever else related after the rebase, makes no sense now

**Review by drishu — CHANGES_REQUESTED** [review]
Move also the paragraphs references in the README, and oe_bootstrap_theme_helper_requirements

**Review by drishu — CHANGES_REQUESTED** [review]
found field--entity-reference-revisions--paragraph.html.twig

**Review by donquixote — APPROVED** [review]
Looks good to me now.

**Review by drishu — APPROVED** [review]

**Pull request merged**